packages/expo-prevent-screen-capture/android/build.gradle

```diff
@@ -0,0 +1,78 @@
+apply plugin: 'com.android.library'
+apply plugin: 'kotlin-android'
+apply plugin: 'kotlin-android-extensions'
+apply plugin: 'maven'
+
+group = 'host.exp.exponent'
+version = '1.0.0'
+
+buildscript {
+  // Simple helper that allows the root project to override versions declared by this library.
+  ext.safeExtGet = { prop, fallback ->
+    rootProject.ext.has(prop) ? rootProject.ext.get(prop) : fallback
+  }
+
+  repositories {
+    mavenCentral()
+  }
+
+  dependencies {
+    classpath("org.jetbrains.kotlin:kotlin-gradle-plugin:${safeExtGet("kotlinVersion", "1.3.50")}")
+  }
+}
+
+// Upload android library to maven with javadoc and android sources
+configurations {
+  deployerJars
+}
+
+// Creating sources with comments
+task androidSourcesJar(type: Jar) {
+  classifier = 'sources'
+  from android.sourceSets.main.java.srcDirs
+}
+
+// Put the androidSources and javadoc to the artifacts
+artifacts {
+  archives androidSourcesJar
+}
+
+uploadArchives {
+  repositories {
+    mavenDeployer {
+      configuration = configurations.deployerJars
+      repository(url: mavenLocal().url)
+    }
+  }
+}
+
+android {
+  compileSdkVersion 26
+
+  defaultConfig {
+    minSdkVersion 21
+    targetSdkVersion 26
```

Comment:
Shouldn't we use a newer version for `targetSdkVersion` and `compileSdkVersion`? Shouldn't we grab them from `rootProject.ext` if possible? 🤔 I'm not an expert when it comes to this, I think it may be worth to ask @mczernek for an opinion.

Reply:
I'm probably less of an expert on this, but whatever we do decide should be changed in the [template](https://github.com/expo/expo/blob/master/packages/expo-module-template/android/build.gradle#L52-L57)